.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Feature / Bug Description
+Describe the feature / bug. If it's a bug, explain the root cause and how to reproduce.
+
+### Solution
+Describe your changes and explain the technical solution.
```

Comment:
why did you change this?

Reply:
I don't remember changing this file, I think it's from when the PR template didn't load and I had to pull it after the change you made. Since then the template change was added to my comm